src/main/java/banking/database/DBConnector.java

```diff
@@ -0,0 +1,38 @@
+package banking.database;
```

Comment:
Can you rename the package to `manager` or `configuration`?

Reply:
fixed